tools/tpm2_create.c

```diff
@@ -347,10 +347,16 @@ execute_tool (int              argc,
             break;
         case 'I':
```

Comment:
Wouldn't this make passing data to be sealed a required option then? Instead of optional like is now?